src/shared/pipes/object-or-steam-id.pipe.ts

```
@@ -0,0 +1,25 @@
+import { Injectable, PipeTransform, BadRequestException } from '@nestjs/common';
+import { Types } from 'mongoose';
+import * as SteamID from 'steamid';
+
+/**
+ * Throws BadRequestException if the provided value is not a valid ObjectId OR SteamId.
+ *
+ * @export
+ * @class ObjectOrSteamIdPipe
+ * @implements {PipeTransform}
+ */
+@Injectable()
+export class ObjectOrSteamIdPipe implements PipeTransform {
+  transform(value: string) {
+    try {
+      if (Types.ObjectId.isValid(value) || new SteamID(value).isValid()) {
+        return value;
+      }
```

No `else` statement? Both `isValid()` methods will throw instead of returning false?

---

Seems to pass tests without the else, so I'm thinking it doesn't matter.

---

:thinking: